scripts/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [[ -z $(git diff --cached --name-only -r | grep status-go-version.json) ]]; then
+  exit 0 # No status-go-version.json changes found.
+fi
+
+MERGE_BASE=$(git merge-base -a develop "$(git rev-parse --abbrev-ref HEAD)" develop)
+GO_COMMIT_MERGE_BASE=$(git show "$MERGE_BASE":status-go-version.json | jq -r '."commit-sha1"')
+GO_COMMIT_CURRENT=$(jq -r '."commit-sha1"' status-go-version.json)
+
+GITHUB_LINK="https://github.com/status-im/status-go/compare/${GO_COMMIT_MERGE_BASE}...${GO_COMMIT_CURRENT}"
+
+#todo remove this echo
+echo "$GITHUB_LINK"
+
+COMMIT_MSG_FILE=$1
+if [[ -f "$COMMIT_MSG_FILE" ]]; then
+  #todo edit link if exists
+  COMMIT_MSG_FILE_CONTENT=$(cat "$COMMIT_MSG_FILE")
+  echo "$COMMIT_MSG_FILE_CONTENT"
+  echo "${GITHUB_LINK}" >>"${COMMIT_MSG_FILE}"
```

Comment:
Something like this might work:
```sh
#!/usr/bin/env bash
sed -i -n -e '/^TEST=/!p' -e '2 iTEST=text'$"\n" $1
```
Which gives me:

![image](https://user-images.githubusercontent.com/2212681/156736415-982248a1-f428-456a-8090-c8ec0da111e0.png)

On first commit. You'll just have to adjust it to use part of github URL as prefix.

Reply:
Also, I don't think the first argument with the message path is always there, so no need to check.